x-pack/plugins/snapshot_restore/public/application/components/repository_form/type_settings/azure_settings.tsx

```diff
@@ -61,6 +62,19 @@ export const AzureSettings: React.FunctionComponent<Props> = ({
     });
   };
 
+  const disableToolTip = (component: ReactElement): ReactElement => {
```

Comment:
Since this logic is used in multiple places, can we create a new React component under `snapshot_restore/public/application/components`  that takes `component` and `isManagedRepo` props and returns the corresponding component?

Reply:
Thanks, I created a dedicated React component called DisableTooltip for the managed repository.